chalice/deploy/validate.py

```diff
@@ -23,6 +51,18 @@ def validate_configuration(config):
     _validate_manage_iam_role(config)
     validate_python_version(config)
     validate_unique_function_names(config)
+    validate_feature_flags(config.chalice_app)
+
+
+def validate_feature_flags(chalice_app):
```

Comment:
Do we want to have some sort of warning/alert that something has been removed from the feature flag list and is now a stable feature? Looks like it doesn't _really_ matter but if I upgrade chalice and redeploy and blueprints was moved to stable. I will still have my `BLUEPRINTS` entry in the experimental feature flags list when not necessary.

Reply:
+1, though it doesn't necessarily need to be added now since we won't be using it yet.

Reply:
I'd prefer not to.  I think warnings should only be for things that may cause issues for a user (i.e. a deprecation of something).   In this case it's just unnecessary code that runs.

Perhaps in the future we can have a some sort of linting command that can check these things, but I don't think it needs to happen at deploy time.

Reply:
Linting command seems good. Agreed it doesn't need to be deploy time.